chef_master/source/resource_execute.rst

```diff
@@ -399,7 +399,7 @@ The following examples demonstrate various approaches for using resources in rec
      command 'slapadd < /tmp/something.ldif'
      creates '/var/lib/slapd/uid.bdb'
      action :run
-     environment ({'HOME' => '/home/myhome'})
+     environment('HOME' => '/home/myhome')
```

Comment:
Should probably skip the parens entirely since we leave them off all other properties.

Reply:
I find this to be a bit more readable and explicit, especially if you break out into multiline, like this:
```ruby
environment(
  HOME: '/home/coderanger',
  PATH: '/usr/bin:/usr/bin/local'
)
```